.github/workflows/buildAndroid.yml

```diff
@@ -71,6 +71,18 @@ jobs:
       - name: Overwrite plugin engine version
         run: |
           ((Get-Content -path CesiumForUnreal.uplugin -Raw) -replace '"EngineVersion": "5.1.0"','"EngineVersion": "${{ inputs.unreal-engine-version }}"') | Set-Content -Path CesiumForUnreal.uplugin
+      - name: Customize BuildConfiguration.xml
+        run: |
+          mkdir -p "$env:USERPROFILE\AppData\Roaming\Unreal Engine\UnrealBuildTool"
+          # - We limit parallel actions because our builds use a lot more memory than UBT thinks they will.
+          # - We set the source code control Provider to None so UBT includes all files in the unity build.
+          Set-Content -Path "$env:USERPROFILE\AppData\Roaming\Unreal Engine\UnrealBuildTool\BuildConfiguration.xml" -Value '<?xml version="1.0" encoding="utf-8" ?>
+            <Configuration xmlns="https://www.unrealengine.com/BuildConfiguration">
+              <BuildConfiguration>
+                <MaxParallelActions>2</MaxParallelActions>
```

Comment:
Just curious, is setting `MaxParallelActions` just a safety measure?

It wasn't obvious to me that this had any impact, at least on performance

Reply:
Some of the platforms need that in order to avoid exhausting their memory and running really slow (by swapping) or crashing. But I probably got overzealous copying it everywhere.